stress/shell/src/main/java/alluxio/stress/cli/Benchmark.java

```diff
@@ -87,10 +87,12 @@ protected static void mainInternal(String[] args, Benchmark benchmark) {
    * @return the JobConfig
    * */
   public PlanConfig generateJobConfig(String[] args) {
-    // remove the cluster flag
+    // remove the cluster flag and java opts
```

Comment:
The java opts are intentionally passed to the spawned tasks, so that the tasks can also be started with a larger mem size

Reply:
when are the java opts used?